spec/support/request_helpers.rb

```diff
@@ -135,4 +135,17 @@ def check_forwards_locale_extension
       end
     end
   end
+
+  def check_accepts_root_path
+    context "with the root path as a base_path" do
+      let(:base_path) { "/" }
+
+      it "creates the content item" do
+        put_content_item
+
+        expect(response.status).to eq(200)
+        expect(a_request(:put, %r{.*/(content|publish-intent)/$})).to have_been_made.at_least_once
```

Comment:
I guess this regexp is to accomodate the possibility that it's either a draft or a live put request. I was a bit confused by this at first. I wonder if it would be sufficient to check the assigns in the controller given that other tests verify that the correct thing is passed through to content store?